astronomer/providers/amazon/aws/sensors/redshift_cluster.py

```diff
@@ -36,7 +45,7 @@ def execute(self, context: Context) -> None:
                 aws_conn_id=self.aws_conn_id,
                 cluster_identifier=self.cluster_identifier,
                 target_status=self.target_status,
-                polling_period_seconds=self.poll_interval,
+                polling_period_seconds=self.poke_interval,
```

Comment:
```suggestion
                poke_interval=self.poke_interval,
```
Let's make this variable name consistent across the repo.

Reply:
Fixed